server/auth.go

```diff
@@ -28,10 +28,11 @@ import (
 	"sync/atomic"
 	"time"
 
+	"golang.org/x/crypto/bcrypt"
+
```

Comment:
Remove newline here.

Reply:
I'm using the `goimports -local github.com/nats.io/` style where it [groups our project imports separately from stdlib and everything else](https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package).

Do you specifically just want the stdlib and everything else (including our stuff) style?

Reply:
ok usually we put these at bottom with a single line space.